File: src/main/java/com/onmeet/team/repository/TeamRepository.java
package com.onmeet.team.repository;

import com.onmeet.team.entity.Team;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface TeamRepository extends JpaRepository<Team, String>, TeamRepositoryCustom {
    List<Team> findByCompanyIdOrderByNameAsc(String companyId);
}

File: auth-service/src/main/kotlin/com/onmeet/auth/repository/jpa/TeamRepository.kt
package com.onmeet.auth.repository.jpa

import com.onmeet.auth.entity.Team
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface TeamRepository : JpaRepository<Team, Long> {
    fun findByNameAndCompanyId(name: String, companyId: Long): Team?
    fun findAllByCompanyId(companyId: Long): List<Team>
}
